src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -323,7 +318,7 @@ public Node call() throws Exception {
                 }
 
                 return slave;
-            } catch (Exception ex) {
+            } catch (Throwable ex) {
```

Comment:
I got strange low level issues while experimenting so preferred to catch all throwable so they get logged